tests/ad/event/test_event_api.py

```diff
@@ -0,0 +1,74 @@
+'''API tests for Event APIs'''
+import datetime
+import responses
+from tests.ad.conftest import RE_BASE
+
+
+@responses.activate
+def test_event_details(api):
+    '''test for details method response'''
+    responses.add(responses.GET,
```

Comment:
Would testing with cassettes ([pytest-vcr](https://pytest-vcr.readthedocs.io/en/latest/)) be beneficial instead of the `responses` library?